core/proto/accountmanager/v1/accountmanager.proto

```diff
@@ -0,0 +1,156 @@
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+option go_package = "github.com/google/keytransparency/core/proto/accountmanager/v1;accountmanager_proto";
+
+package accountmanager.v1;
+
+import "google/api/annotations.proto";
+import "google/protobuf/field_mask.proto";
+import "proto/keytransparency_v1_proto/keytransparency_v1_proto.proto";
+import "proto/keymaster_proto/keymaster_proto.proto";
+
+// GetKeySetsRequest requests the keyset of a domain_id/app_id 
+message GetKeySetRequest {
+   // domain_id identifies the domain.
+   string domain_id = 1;
+   // app_id identifies the application.
+   string app_id = 2;
+}
+
+// CreateUserRequest specifies the information with which a new account should be initialized.
+// New accounts will be signed with the current active key.
+// It is the responsibility of authorized callers to verify that domain_id/app_id/user_id is correct.
+message CreateUserRequest {
+  // account is the account to create.
+  keytransparency.v1.proto.User account = 2;
+  // add_signing_keys specifies whether to add this service's signing keys to the set of authorized_keys.
+  // This must be set to true if any further operations from this API are meant to succeed. 
+  // If set to false, there must be at least one key in authorized_keys.
+  bool add_signing_keys = 7;
+}
+
+// UpdateUserRequest sets the data field for the account.
+// The account must have the service's current signing key in its list of
+// authorized_keys in order to succeed.
+message UpdateUserRequest {
+  // account contains data which will be applied to the account.
+  keytransparency.v1.proto.User account = 2;
+  // update_mask specifies which fields of account to update.
+  // For example: "data" or "authorized_keys"
+  google.protobuf.FieldMask update_mask = 3;
+}
+
+// BatchCreateUserRequest creates multiple accounts all at once.
+message BatchCreateUserRequest {
+   // domain_id identifies the domain.
+   string domain_id = 1;
+   // app_id identifies the application.
+   string app_id = 2;
+  // accounts is the set of accounts to create.
+  repeated keytransparency.v1.proto.User accounts = 3;
+  // add_signing_keys specifies whether to add this service's signing_keys to the set of authorized_keys.
+  // This must be set to true if any further operations from this API are meant to succeed. 
+  // If set to false, there must be at least one key in authorized_keys.
+  bool add_signing_keys = 4;
+}
+
+// BatchCreateUserRequest creates multiple accounts at once.
```

Comment:
Please fix the comment to reflect the object name.

Reply:
done